adserver/forms.py

```diff
@@ -156,10 +163,14 @@ def clean(self):
 
             # Check text length
             if ad_type.max_text_length:
-                stripped_text = bleach.clean(text, tags=[], strip=True)
+                if text:
+                    stripped_text = bleach.clean(text, tags=[], strip=True)
+                else:
+                    stripped_text = f"{headline} {body} {cta}"
```

Comment:
This feels a bit weird. We should probably normalize the spaces to make sure it only includes 1 space.

Reply:
When the ad is actually rendered, this does happen correctly. This is only used to check that the ad length isn't too long.